Simulation/6DOF_RK4/Executables/analysis/monte_carlo_plotter_plotly.py

```diff
@@ -0,0 +1,156 @@
+import numpy as np
+import pandas as pd
+import os
+import sys
+import plotly.graph_objects as go
+from plotly.graph_objs import Mesh3d
+
+sys.path.insert(0, os.path.abspath(
+    os.path.join(os.path.dirname(__file__), '..','..','Configurations')))
+
+import data_loader as dataloader
+config = dataloader.config
+
+# Indices of each state variable in the output array
+# kf and rkf values are arrays containing position, velocity and acceleration in each axis
+indices = {"time":0,
```

Comment:
better as an enum, see [enum package](https://docs.python.org/3/library/enum.html)

Reply:
also looking back, probably better to just have a global dataframe or smth to hold sim data because then we can access fields by property name and it's alr sorted by timestamps